lib/cell.rb

```diff
@@ -20,7 +20,7 @@ def fired_upon?
   end
 
   def fire_upon
-    @ship.hit if @ship && !fired_upon?
```

Comment:
I removed this check because we are checking if the cell has been fired upon elsewhere